### PR TITLE
Kochnev/mailings gallery

### PR DESCRIPTION
**Pull request by rosendi**

### CODE REVIEW

File: src/blocks/wizard/__step/DraftEditContent/Preview/Overlay.ejs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="wizard__stepOverlay">
+    <div class="wizard__stepOverlayButton"><%= UI.icons.edit %>&nbsp;<%- i18n('Нажмите чтобы отредактировать') %></div>
```
Comment (astupak) [inline src/blocks/wizard/__step/DraftEditContent/Preview/Overlay.ejs]: UI

File: src/blocks/wizard/__step/DraftEditContent/index.ejs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    var Footer = require('blocks/shared/WizardEditorFooter/index');
+    var DraftEditContentActions = require('./DraftEditContentActions');
+    var Preview = require('./Preview/index');
+%>
+
+<div class="<%- self.generateClassName('wizard__step', self.modifiers) %>">
+    <div class="wizard__stepTitle js-toggle-step">
+        <div class="wizard__stepTitleIndex">
+            <div class="wizard__stepTitleIndexNumber"><%- self.index %></div>
+            <div class="wizard__stepTitleIndexCheck"><%= UI.icons.circleCheck %></div>
```
Comment (astupak) [inline src/blocks/wizard/__step/DraftEditContent/index.ejs]: UI

File: src/blocks/wizard/__step/DraftEditContent/Preview/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    return Block.extend({
+        template: template,
+
+        events: {
+            'click .js-overlay': '_onOverlayClick'
+        },
+
+        _onOverlayClick: function() {
+            if(this.props.onClick) {
```
Comment (zouxuoz) [inline src/blocks/wizard/__step/DraftEditContent/Preview/index.js]: missing whitespace after `if`

File: src/blocks/wizard/__step/DraftEditContent/EditTemplate/EditorActions.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                    formModel: self.props.draft,
+                    options: {
+                        text: self.props.template.compile(),
+                    },
+                    title: i18n('Отправка тестовой копии'),
+                    formParams: {
+                        action: 'issue.send',
+                        modal: self
+                    },
+                    cancel: function(){
```
Comment (zouxuoz) [inline src/blocks/wizard/__step/DraftEditContent/EditTemplate/EditorActions.js]: missing whitespace before `{`

**Comment by zouxuoz** [comment]
LGTM 👯

**Pull request merged**